webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -0,0 +1,279 @@
+        }
+
+        return (<span className='ticket-status-default default-style'>{ticketStatus}</span>);
+    }
+
+    labelList(labels: string[]) {
+        if (labels.length) {
+            let totalString = 0;
+            let totalHide = 0;
+            const labelList = labels.map((label: string, key: number): JSX.Element => {
+                if (totalString < 3) {
+                    totalString++;
+                    return (
+                        <span
+                            key={key}
+                            className='jiraticket-popover-label-list'
+                        >
+                            {label}
+                        </span>);
+                }
+                totalHide++;
+                if (key === labels.length - 1) {
+                    const moreLabels = `+${totalHide}more`;
+                    return (
+                        <span
+                            key={key}
+                            className='jiraticket-popover-label-list'
+                        >
+                            {moreLabels}
+                        </span>);
+                }
+                return <></>;
+            });
+
+            return (<div className={'ticket-popover-label'}>{labelList}</div>);
+        }
+        return null;
+    }
+
+    render() {
+        if (!this.state.isLoaded) {
+            return (<p/>);
+        }
+        const ticketDetails = this.state.ticketDetails;
+        const jiraTicketURI = this.state.href;
+        const jiraAvatar = ticketDetails && ticketDetails.jiraIcon;
+        const jiraIssueIconURI = ticketDetails && ticketDetails.issueIcon;
+        const jiraTicketKey = ticketDetails && ticketDetails.ticketId;
+        const jiraTicketTitle = ticketDetails && ticketDetails.summary;
+        const jiraTicketAssigneeAvatarURI = ticketDetails && ticketDetails.assigneeAvatar;
+        const jiraTicketAssigneeName = ticketDetails && ticketDetails.assigneeName;
+        const jiraTicketStatusName = ticketDetails && ticketDetails.statusKey;
+        const jiraTicketDescription = ticketDetails && ticketDetails.description;
+        const jiraTicketVersions = ticketDetails && ticketDetails.versions;
+        const jiraTicketLabels = ticketDetails && ticketDetails.labels;
+        const isAssigned = ' is assigned';
+        const unAssigned = 'Unassigned';
+
+        return (
+            <div className={'ticket-popover'}>
+                <div className={'ticket-popover-header'}>
+                    <div className={'ticket-popover-header-container'}>
+                        <a
+                            href={jiraTicketURI}
+                            title={'goto ticket'}
+                        >
+                            <img
+                                src={jiraAvatar}
+                                width={14}
+                                height={14}
+                                alt={'jira-avatar'}
```

Comment:
```suggestion
                                alt='jira-avatar'
```
When passing string in props, I don't think there's a need for brackets. Fix this everywhere.